server/scratchy_server/resources/messageres.py

```diff
@@ -13,14 +15,22 @@ def get(self, messageId=None):
             roomId = request.args.get('roomid')
             if roomId:
                 try:
-                    return Response(MessageModel.objects.filter(roomId=roomId).to_json(), mimetype="application/json", status=200)
+                    response = Response(MessageModel.objects.filter(roomId=roomId).to_json(), mimetype="application/json", status=200)
                 except MessageModel.DoesNotExist as ie:
                     abort(404)
+                else:
+                    logging.debug("here are the message:")
+                    for i in json.loads(response.get_data()):
```

Comment:
Voir pour remplacer le for par un `map` qui retourne les contents de chaque message.
Entourer le tout d'un if qui s'exécute que quand on est en debug pour éviter d'exécuter du code qui "sert a rien" quand on est pas en debug.

Comme ceci  : `logging.getLogger().isEnabledFor(logging.DEBUG)`